docs/APISocketOptions.md

```diff
@@ -124,13 +124,16 @@ their characteristics according to the following legend:
 is empty, it's an option derived from UDT. "Version 0.0.0" is the oldest version
 of SRT ever created and put into use.
 
-2. **Binding**: Defines limitation on setting the option. The field is empty if the option
+2. **Restrict**: Defines restrictions on setting the option. The field is empty if the option
 is not settable (see **Dir** column):
 
     - `pre`: A connecting socket (both as caller and rendezvous) must be set
-prior to calling `srt_connect()` or `srt_bind()` and never changed thereafter.
-A listener socket should be set to "listening" and it will be
-derived by every socket returned by `srt_accept()`.
+prior to calling `srt_connect()` and never changed thereafter. A listener
+socket should be set to "listening" and it will be derived by every socket
+returned by `srt_accept()`.
+
+    - `pre-bind`: Like `pre`, but the option is not allowed to be altered after
+calling `srt_bind()`.
```

Comment:
Maybe some note would be useful here, mentioning that `srt_connect()` will bind implicitly even if `srt_bind(..)` was not called.
Or:
"Like `pre`, but additionally the option is not allowed to be altered after calling `srt_bind()`, whichever happens first."

Reply:
Maybe even "pre-bind" should go before "pre" as more restrictive.

Reply:
Maybe it's more restrictive, but OTOH it's less common among options. Mentioning this bind-when-connect might be a good idea.